R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -1327,6 +1327,13 @@ test_that("SQL error message is returned from JVM", {
   expect_equal(grepl("Table Not Found: blah", retError), TRUE)
 })
 
+test_that("Method as.data.frame as a synonym for collect()", {
+  irisDF <- createDataFrame(sqlContext, iris)
+  expect_equal(as.data.frame(irisDF), collect(irisDF))
+  irisDF2 <- irisDF[irisDF$Species == "setosa", ]
+  expect_equal(as.data.frame(irisDF2), collect(irisDF2))
```

Comment:
I believe this is not necessary, since when I declared setGeneric("as.data.frame"), I didn't specify any definition. Therefore, my implementation of as.data.frame would not overwrite R's. I tested it myself, and as.data.frame() still works for objects other than DataFrame.